src/utils/utils.ts

```diff
@@ -223,6 +223,12 @@ function enableDraggingWithScaling(element: HTMLElement): void {
 async function onElementDropComplete(dragElement: HTMLElement, dropElement: HTMLElement): Promise<void> {
   if (!dropElement) return;
 
+
+  const container = document.getElementById('container');
+  const objectiveString = container.getAttribute('objective');
+  const showCheck = container.getAttribute('showCheck');
+  const isContinueOnCorrect = container.getAttribute('isContinueOnCorrect');
```

Comment:
isContinueOnCorrect and inCorrect should be executed even in different templets like  Multi option question not just Drag and drop

Add The login in the OnActivityComplete